T/Telescope/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Telescope"
+version = v"0.1.2"
+
+# Collection of sources required to build this package
+sources = [
+    ArchiveSource("https://github.com/jhigginbotham64/Telescope/archive/refs/tags/v$(version).tar.gz",
+              "025a24a5dfe4c4d5910ae0cad7664ec2c2fc21cce8ffaae56cc6b42300249de6"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+export CFLAGS="-I${prefix}/include"
+export CXXFLAGS="-I${prefix}/include"
+cd $WORKSPACE/srcdir/Telescope*
+mkdir build && cd build
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -S .. \
+    -B .
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl")
+]
```

Comment:
Looks like this can be simplified to
```suggestion
platforms = supported_platforms(; exclude=p->!(Sys.islinux(p) && nbits(p) == "64"))
```
What are the problems on the other platforms?  Maybe they can be fixed, at least partially?  However the decision of the platforms to support is up to you

Reply:
Incorporating the simplification seems to lead to an empty platform list.

As to your question, I described them briefly [here](https://github.com/JuliaPackaging/Yggdrasil/pull/4621#issuecomment-1068485860) and failed pipeline outputs are visible [here](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=18263&view=logs&j=982d096e-6cad-5609-3037-305d94247f4e&t=f7dedcfd-9020-5606-352e-77d182153651&l=366).

Reply:
That's because I got the value of `nbits(p)` wrong: it should have been the number `64`, not the string `"64"`